lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -466,3 +467,62 @@ def upload_ora2_submission_files(
     TASK_LOG.info('%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def generate_anon_ids(_xmodule_instance_args, _entry_id, course_id, task_input, action_name):  # lint-amnesty, pylint: disable=too-many-statements
+    """
+    Generate a 2-column CSV output of user-id, anonymized-user-id
+    """
+
+    start_time = time()
+    start_date = datetime.now(UTC)
+
+    fmt = 'Task: {task_id}, InstructorTask ID: {entry_id}, Course: {course_id}, Input: {task_input}'
+    task_info_string = fmt.format(
+        task_id=_xmodule_instance_args.get('task_id') if _xmodule_instance_args is not None else None,
+        entry_id=_entry_id,
+        course_id=course_id,
+        task_input=task_input
+    )
+    TASK_LOG.info('%s, Task type: %s, Starting task execution', task_info_string, action_name)
```

Comment:
This should be the first line in the method.

Reply:
I would suggest having a without formatted simple log at the 1st line of the task, this will make sure our celery task is executed. This will be a temporary log & can be removed after testing.